packages/venia-concept/src/components/ProductOptions/options.js

```diff
@@ -1,16 +1,14 @@
 import React, { Component } from 'react';
-import { arrayOf, func, shape, string } from 'prop-types';
+import { func, object } from 'prop-types';
+
+import isProductConfigurable from 'src/util/isProductConfigurable';
 
 import Option from './option';
 
 class Options extends Component {
     static propTypes = {
         onSelectionChange: func,
-        options: arrayOf(
-            shape({
-                attribute_id: string.isRequired
-            })
-        ).isRequired
+        product: object
```

Comment:
Instead of taking `options`, this `Options` component now takes a `product` prop. It internally handles whether to show options based on whether the product is configurable or not.

This allowed us to remove a lot of repeated conditional logic from other components.

Reply:
I shall double check this, good question 👍